build.xml

```diff
@@ -28,15 +28,12 @@
   <property name="compile.java.bootclasspath" value="${build.bootclasspath}"/>
 
   <path id="adaptor.build.classpath">
-<!--
```

Comment:
Putting jdbc jar in lib would make it errorprone to accidently commit that jar.  Thanks for showing this interpretation. It wasn't intended.